hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -507,7 +511,7 @@ prop_canCloseFanoutEveryCollect = monadicST $ do
     _ -> fail "not observed close"
   -- Fanout
   let fanoutUTxO = getKnownUTxO stClosed
-  let txFanout = unsafeFanout cctx fanoutUTxO seedTxIn initialUTxO Nothing (slotNoFromUTCTime systemStart slotLength deadline)
+  let txFanout = unsafeFanout cctx fanoutUTxO seedTxIn initialUTxO Nothing Nothing (slotNoFromUTCTime systemStart slotLength deadline)
```

Comment:
yet again we miss to test fanout with something to increment/decrement

Reply:
Here we are closing with initial snapshot so both utxo to commit and decommit are expected to be Nothing